Quantum-Control-Applications-QuAM/Superconducting/configuration/make_quam.py

```diff
@@ -81,11 +85,14 @@ def create_quam_superconducting(
             )
             machine.octaves[f"octave{i+1}"] = octave
             octave.initialize_frequency_converters()
+            # TODO: what is quam.octave?
             print("Please update the octave settings in: quam.octave")
 
     # Add the transmon components (xy, z and resonator) to the quam
     for qubit_name, qubit_wiring in machine.wiring.qubits.items():
         # Create all necessary ports
+        # TODO: hide it from the customer + what does it do?
+        # TODO: why not adding the port directly  when building the base components?
```

Comment:
I thought to put everything here because if it's together with the rest it'll be even more confusing.
I agree it looks rather complicated, what about we create a separate function `create_ports(qubit_wiring)`?